tests/Bridge/Symfony/Bundle/Command/SwaggerCommandTest.php

```diff
@@ -19,6 +19,7 @@
 
 /**
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ * @group legacy
```

Comment:
Just a lot of deprecations because of the UnderscoreOperationResolver class deprecation.